src/frontend/src/flows/verifiableCredentials/allowCredentials.ts

```diff
@@ -35,6 +41,15 @@ const allowCredentialsTemplate = ({
     onSubmit: (userNumber) => onAllow(userNumber),
   });
 
+  const consentMessage = new Chan<TemplateElement>(html`${consentMessage_}`);
+
+  // Kickstart markdown parsing & sanitizing; once done, replace the consent message
+  void (async () => {
+    const parsed = await parseMarked(consentMessage_);
```

Comment:
in NNS dapp, @mstrasinskis has implemented various functions before and at the top of marked to either provide more security (e.g. pre-escape svg or render images as links) or tweak rendering (e.g. render link with target blank). Would those be useful here?

source: https://github.com/dfinity/nns-dapp/blob/4619857a316be5b47a950c6b8461ffcaa1a3dde3/frontend/src/lib/utils/html.utils.ts#L94

Reply:
nice, let me have a look!

Reply:
Included, thanks 👍